apps/console/src/features/console-settings/components/console-roles/create-console-role-wizard/create-console-role-wizard.tsx

```diff
@@ -36,7 +36,9 @@ import { CreateRoleInterface, CreateRolePermissionInterface } from "../../../../
 import { ConsoleRolesOnboardingConstants } from "../../../constants/console-roles-onboarding-constants";
 import useConsoleSettings from "../../../hooks/use-console-settings";
 
-export type CreateConsoleRoleWizardPropsInterface = IdentifiableComponentInterface & ModalProps;
+export interface CreateConsoleRoleWizardPropsInterface extends IdentifiableComponentInterface, ModalProps {
+    onClose: (e: MouseEvent<HTMLElement>, data: ModalProps) => void;
```

Comment:
Isn't `onClose` inherited from `ModalProps` ?

Reply:
Eventhough this is inherited from `ModalProps` the es-lint was complaining about missing proptypes. That's why I explicitly added the prop
<img width="1011" alt="Screenshot 2024-01-18 at 11 22 30" src="https://github.com/wso2/identity-apps/assets/42619922/2f00e3b8-264b-4218-a57f-343b9ca0297d">